src/main/scala/com/typesafe/sbt/packager/archetypes/systemloader/SystemloaderPlugin.scala

```diff
@@ -50,6 +51,7 @@ object SystemloaderPlugin extends AutoPlugin {
   def systemloaderSettings: Seq[Setting[_]] = Seq(
     serverLoading := None,
     serviceAutostart := true,
+    fileDescriptorLimit := Some("1024"),
```

Comment:
I think this should go into the `LinuxPlugin`. At somepoint I will refactor this into the `SystemloaderPlugin` where it belongs, but for this PR it's fine to put it into the `LinuxPlugin`